docs/docs/markers.mdx

```diff
@@ -0,0 +1,291 @@
+---
+id: markers
+sidebar_label: Markers
+title: Markers
+description: Read more about how to mark points of interest in dialogues using logical expressions.
+abstract: Markers are logical expressions that allow you to describe points in the dialogue that you are interested in. 
+---
+
+:::caution
+
+This feature is currently experimental and might change or be removed in the future. Share your feedback in the forum to help us make it production-ready.
+
+:::
+
+Markers allow you to describe points in a dialogue that you are interested in identifying. In Rasa, a dialogue is represented as a sequence of events, which include bot actions that were executed (`ActionExecuted`), intents that were detected (`UserUttered`), and slots that were set (`SlotSet`). A point of interest in a dialogue can thus be expressed as a logical expression that can be evaluated against the sequence of dialogue events. When this logical expression is met, we "mark" that point in the dialogue for further analysis or inspection.
+
+There are several applications for Markers. For example, they can be used to define your bot's Key Performance Indicators (KPIs), such as dialogue completion or task success. For [Carbon Bot](https://rasa.com/blog/using-conversation-tags-to-measure-carbon-bots-success-rate/) (which helps users offset their carbon emissions from flying) dialogue completion can be defined as "all mandatory slots have been filled", while task success can be defined as "all mandatory slots have been filled and a carbon estimate has been successfully computed". Marking these events allows us to evaluate Carbon Bot's performance by quantifying how often it succeeds and how often it fails.
```

Comment:
One thing that has come up in discussions with customers again and again is that they like to measure containment (by which they usually mean dialogue completion) and track human-handoff. It might make sense to add an example around human-handoff in the docs as this is something that will a problem that is immediately familiar to them when they read the docs.

Reply:
> "Marking these events allows us to evaluate Carbon Bot's performance by quantifying how often it succeeds and how often it fails."

Since the word "performance" can mean many things and may be confused with machine learning stats. Does it make sense to this one to: 

> "Marking these events allows us to track how often conversations in Carbon Bot cause the effect it's designed for."